src/wallet/sqlite.cpp

```diff
@@ -365,6 +365,14 @@ void SQLiteBatch::Close()
     }
 }
 
+// disable -Wzero-as-null-pointer-constant due to SQLITE_STATIC
```

Comment:
If I remove changes in this file I won't receive warning with both gcc 9.3.0 and clang 10.0.0.
What cases are guarded here?

Having system sqlite 3.31.1.